.changeset/weak-apes-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/node": minor
```

Comment:
Shouldn't this be a `major` because it uses the new integration `astro/app` APIs?

Reply:
You're right. I'll change that.